hdfs-connector/src/main/java/com/oracle/bmc/hdfs/BmcProperties.java

```diff
@@ -46,6 +46,13 @@ public enum BmcProperties {
      * Incompatible with READ_AHEAD.
      */
     IN_MEMORY_READ_BUFFER(IN_MEMORY_READ_BUFFER_KEY, false),
+
+    /**
+     * (boolean, optional) Flag to disable the SDK level circuit breaker.
+     * See {@link BmcConstants#SDK_CIRCUIT_BREAKER_DISABLE_KEY} for config key name. Default is false.
+     */
+    SDK_CIRCUIT_BREAKER_DISABLE(SDK_CIRCUIT_BREAKER_DISABLE_KEY, false),
```

Comment:
Suggestion: use this key:
OBJECT_STORAGE_CLIENT_CIRCUIT_BREAKER_ENABLED, default true

Reply:
Thanks . Resolved !